basic-skills/docs/basic-daemon/README.md

```diff
@@ -0,0 +1,94 @@
+# basic-daemon Daemon Skill (UPDATE THIS FOR YOUR SKILL)
+
+A system generated Skill based on the custom daemon Skill template. 
+
+> Modify these sections for your specific use case.
+
+## Files Generated
+- `docs/` - The directory that houses the Skills' READMEs
+  - `basic-daemon/`: The directory that houses the basic-daemon Skill's README
+    - `README.md`: Provide the objectives, requirements, and instructions for generating and deploying the Skill here.
+- `skills/`: The directory that houses the Skills
+  - `basic-daemon/`: The directory that houses the basic-daemon Skill's assets
+    - `actions/`: The directory that houses the basic-daemon Skill's Actions
+      - `basic-daemon/`: The contents of the basic-daemon action
+        - `Dockerfile`: Modify this to build the Docker image the action
+        - `main.py`: Modify the code in this file for your Cortex daemon
+        - `requirements.txt`: List all the dependencies and libraries needed for this Skill
+    - `invoke/`: Contains the payloads, organized by Skill input name, used to invoke the basic-daemon Skill
+      - `request/`: Contains payload files used to invoke the Skill
+        - `message.json`: Write a test JSON payload to invoke the Skill
+      - `skill.yaml`: Define basic-daemon Skill definition and map Actions here
+
+
+## Generate the Skill
+
+You've already done this via:
```

Comment:
This line is worded a bit weird. If they've already done this step at this point maybe change it to "The Skill should have been generated previously. If not..."